### DISCUSSION TITLE
erreur sur la partie 1 & 4

### DISCUSSION THREAD

**Discussion by lucile98-ode** [lucile98-ode/GPI-L2BHA-FENOUIL#21] [Show and tell]
Je détecte que ce code pose problème. Selon la mauvaise pratique (BAD_PRACTICE) , Nm: Les noms de méthode doivent commencer par une lettre minuscule (NM_METHOD_NAMING_CONVENTION) .
Les méthodes doivent être des verbes, en casse mixte avec la première lettre minuscule, avec la première lettre de chaque mot interne en majuscule.

**Comment by chrabou** [comment]
@amrx12 tu pourrais voir stp

**Reply by amrx12** [reply]
pourtant les noms des méthodes (concernant la partie bdd du code) sont en minuscule, seul les méthodes static de l'interface graphique commencent par une majuscule
